apps/challenges/models.py

```diff
@@ -37,6 +37,7 @@ def __init__(self, *args, **kwargs):
         default=False, verbose_name="Publicly Available", db_index=True)
     enable_forum = models.BooleanField(default=True)
     forum_url = models.URLField(max_length=100, blank=True, null=True)
+    leaderboard_description = models.TextField(null=True, blank=True)
```

Comment:
@vkartik97 Can you please mention in comment the default character limit for `TextField`.

Reply:
@RishabhJain2018 
`models.TextField` have no character limit if not defined. It uses Postgres' Text type which also has no character limit. But
>In any case, the longest possible character string that can be stored is about 1 GB. 
- https://www.postgresql.org/docs/9.6/datatype-character.html
- https://docs.djangoproject.com/en/1.11/ref/models/fields/#textfield

Should I add `max_length` and set it to `2048`?
The Description in screenshot has 602 Characters.

Reply:
okay. I think we can leave it as it is.